pnpm-workspace.yaml

```diff
@@ -5,6 +5,7 @@ packages:
   # sample code
   - 'sample-code'
   # exclude packages that are inside test directories
+  - '!tests/e2e-tests' # including the e2e tests makes causes pnpm to explode
```

Comment:
excluding this here has the disadvantage that the e2e tests are not compiled with the rest of the code and one always has to run `pnpm i` before running e2e tests.

But if I include it I end up with around 1500 new files that usually should end up in `.dist`, but are instead mixed into the source code, so it seems to mess with src vs. dist directories..